### PR TITLE
renames

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

**Comment by rgbkrk** [comment]
I linked this repo to travis even though it didn't have tests. You should change our npm test command or put in a dummy test. :wink:

**Comment by jdfreder** [comment]
Look at all of these awesome red Xs.  Woot!  I think the last one will pass....

**Comment by jdfreder** [comment]
.... still haven't moved to iojs.  So can't test locally.  I'll do that tonight. :stuck_out_tongue:

**Comment by rgbkrk** [comment]
> .... still haven't moved to iojs. So can't test locally. I'll do that tonight. :stuck_out_tongue:

That jsdom! Sorry.

File: package.json [hunk 1]
```diff
@@ -41,6 +41,9 @@
   },
   "dependencies": {
     "ansi-to-html": "^0.3.0",
-    "transformime-commonmark": "0.0.1"
+    "transformime-commonmark": "0.0.1",
+    "mocha": "^2.2.5",
```
Comment (rgbkrk) [inline package.json]: Gah! These should be in `devDependencies`. To add in, run `npm install --save-dev mocha`, etc.

I already merged this as is.

**Pull request merged**